src/cosmos/useJsonRpcEndpoint.ts

```diff
@@ -35,5 +35,5 @@ export default function useJsonRpcEndpoint() {
   return useOption('jsonRpcEndpoint', {
     options: endpoints,
     defaultValue: INFURA_NETWORK_URLS[SupportedChainId.MAINNET] ? SupportedChainId[SupportedChainId.MAINNET] : NONE,
-  })
+  })?.at(0)
```

Comment:
What does `at` compile to? If it compiles to `at`, and not `[]`, then prefer to use `[]` directly (as it is more widely available).